features/0023-did-exchange/README.md

```diff
@@ -258,13 +258,17 @@ The _invitee_ will provision a new DID according to the DID method spec. For a P
   "@type": "https://didcomm.org/didexchange/1.0/request",
   "~thread": { "pthid": "<id of invitation>" },
   "label": "Bob",
-  "connection": {
-    "did": "B.did@B:A",
-    "did_doc": {
-        "@context": "https://w3id.org/did/v1"
-      	// DID Doc contents here.
-    }
-  }
+  "did": "B.did@B:A",
+  "did_doc~attach": {
+     "base64": "eyJ0eXAiOiJKV1Qi... (bytes omitted)",
```

Comment:
is the intention to force this form of attachments (base64)?

Reply:
It must be if it is to be signed. Well, any canonical form can be signed, but base64 is the only currently proposed one.

Reply:
can we add a definition of the format of base64 in here? E.g. it's Base64url with no padding (to stay consistent with jose)

Reply:
Any of those improvements should be made to the attachments signing RFC, then reflected here.